lib/Minz/Configuration.php

```diff
@@ -210,14 +210,22 @@ public function __set(string $key, mixed $value): void {
 	 * Save the current configuration in the configuration file.
 	 */
 	public function save(): bool {
+		$tmp_filename = $this->config_filename . '.tmp';
 		$back_filename = $this->config_filename . '.bak.php';
-		@rename($this->config_filename, $back_filename);
 
-		if (file_put_contents($this->config_filename,
+		if (file_put_contents($tmp_filename,
 			"<?php\nreturn " . var_export($this->data, true) . ';', LOCK_EX) === false) {
+			@unlink($tmp_filename);
 			return false;
 		}
 
+		if (copy($this->config_filename, $back_filename) === false) {
```

Comment:
```suggestion
		if (!copy($this->config_filename, $back_filename)) {
```

At least for natively boolean expressions, let's stick to the boolean syntax